pages/dashboard/index.js

```diff
@@ -1,14 +1,48 @@
 const page = require('../../lib/page');
+const moment = require('moment');
 
 module.exports = settings => {
   const app = page({
     ...settings,
     root: __dirname
   });
 
+  const buildTask = (taskCase, req) => {
+    const licence = taskCase.data.model;
+    let action = {};
+
+    switch (licence) {
+      case 'pil':
+        action = {
+          label: 'PIL application',
+          url: req.buildRoute('pil.update', {
+            establishmentId: taskCase.data.establishment.id,
+            profileId: taskCase.data.profile.id,
+            pilId: taskCase.data.id
+          }),
+          details: taskCase.data.profile.name
+        };
+        break;
+    }
+
+    return {
+      receivedAt: moment(taskCase.updated_at).format('D MMM YYYY'),
```

Comment:
hmm, do we have this date format as a constant somewhere? I think we do, if not then we should have a central place as we use formatted dates in a few places. Project list springs to mind

Reply:
Agreed, we should definitely have a standard format across the app. If there isn't one already I think it's a separate ticket :)

Reply:
New ticket: https://jira.digital.homeoffice.gov.uk/browse/AS-769

Reply:
cool, but this line of code is new, so rather than doing it like this and saying "we'll write a ticket to fix them all" wouldn't it be better to have one less magic string? We currently define readable date formate here https://github.com/UKHomeOffice/asl-pages/blob/master/constants/index.js#L2 and we use it in other places